ext_tables.sql

```diff
@@ -26,5 +26,9 @@ CREATE TABLE tx_listelements_item (
 	assets int(11) unsigned DEFAULT '0' NOT NULL,
 	link varchar(1024) DEFAULT '' NOT NULL,
 	linklabel varchar(255) DEFAULT '' NOT NULL,
-	linkconfig varchar(255) DEFAULT '' NOT NULL
+	linkconfig varchar(255) DEFAULT '' NOT NULL,
+
+    KEY tablename (tablename),
+    KEY fieldname (fieldname),
```

Comment:
use one KEY for all three fields (probably because you usually query all three fields at once). In addition, the KEY "name" should not be the same as the fieldname.

e.g. "KEY commonqueryindex (tablename, fieldname, uid_foreign)"

Reply:
👍🏻